platform/features/conductor/src/ConductorRepresenter.js

```diff
@@ -0,0 +1,162 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+define(
+    [],
+    function () {
+        "use strict";
+
+        var CONDUCTOR_HEIGHT = "100px",
+            TEMPLATE = [
+                '<div style=',
+                '"position: absolute; bottom: 0; width: 100%; ',
+                'overflow: hidden; ',
+                'height: ' + CONDUCTOR_HEIGHT + '">',
+                "<mct-include key=\"'time-controller'\" ng-model='conductor'>",
+                "</mct-include>",
+                '</div>'
+            ].join(''),
+            GLOBAL_SHOWING = false;
+
+        /**
+         * The ConductorRepresenter attaches the universal time conductor
+         * to views.
+         *
+         * @implements {Representer}
+         * @constructor
+         * @memberof platform/features/conductor
+         * @param {platform/features/conductor.ConductorService} conductorService
+         *        service which provides the active time conductor
+         * @param $compile Angular's $compile
+         * @param {ViewDefinition[]} views all defined views
+         * @param {Scope} the scope of the representation
+         * @param element the jqLite-wrapped representation element
+         */
+        function ConductorRepresenter(conductorService, $compile, views, scope, element) {
+            var conductorScope;
+
+            // Angular doesn't like objects to retain references to scopes
```

Comment:
...and after a quick check, it doesn't seem necessary in this case. Will change.